backend/app/infrastructure/routes/allocations.py

```diff
@@ -313,8 +300,8 @@ def get(self, proposal_address: str, epoch: int):
             f"Getting donors for proposal {proposal_address} in epoch {epoch}"
         )
         donors = [
-            dataclasses.asdict(w)
-            for w in allocations.get_all_by_proposal_and_epoch(proposal_address, epoch)
+            {"address": w.donor, "amount": str(w.amount)}
```

Comment:
do we have to turn the amount to string? I think it's handled automatically by flask